pkg/codegen/python/gen.go

```diff
@@ -219,7 +219,7 @@ func (mod *modContext) genHeader(w io.Writer, needsSDK, needsJSON bool, imports
 		fmt.Fprintf(w, "import warnings\n")
 		fmt.Fprintf(w, "import pulumi\n")
 		fmt.Fprintf(w, "import pulumi.runtime\n")
-		fmt.Fprintf(w, "from typing import Any, Dict, List, Mapping, Optional, Tuple, Union\n")
+		fmt.Fprintf(w, "from typing import Any, Mapping, Optional, Sequence, Union\n")
```

Comment:
Are we just removing `Tuple` because we don't actually ever use it?

Reply:
And I guess `Dict` was replaced with `Mapping`?

Reply:
Yeah.